cpp/formatting/snippets/AllowShortFunctionsOnASingleLine.cpp

```diff
@@ -1,8 +1,4 @@
-// Only short methods inside a struct or class can be put in a single line
-
-class Foo {
```

Comment:
I would keep the `class Foo {...};` example in case we switch value to `InlineOnly` one day.

Reply:
IMHO the original snippet demonstrates all values `AllowShortFunctionsOnASingleLine` config can take. It encourages discussion if someone wants to propose value `InlineOnly` instead. This is why I would keep it. I will approve the PR. Do as you please.